app/services/articles/feeds/basic.rb

```diff
@@ -9,7 +9,7 @@ def initialize(user: nil, number_of_articles: 25, page: 1, tag: nil)
         @article_score_applicator = Articles::Feeds::ArticleScoreCalculatorForUser.new(user: @user)
       end
 
-      def feed
+      def default_home_feed(**_kwargs)
```

Comment:
This is the related change to above - accept any keyword arguments, and ignore all of them. Because of the two aliases (`feed` and `more_comments_minimal_weight_randomized`) having different method signatures this makes sense.

It's possible other feed strategies will want to ignore unused keywords (and if that's the case, hinting in `LargeForemExperimental#default_home_feed` that all keywords are accepted and ignored might solidify that pattern)?